addons/func_godot/src/netradiant_custom/netradiant_custom_gamepack_config.gd

```diff
@@ -49,6 +49,19 @@ extends Resource
 ## Skip texture path that gets applied to caulk and nodrawnonsolid shaders.
 @export var skip_texture: String = "textures/special/skip"
 
+## Variables to include in the exported gamepack's [code]default_build_menu.xml[/code].[br][br]
+## Each [String] key defines a variable name, and its corresponding [String] value as the literal command-line string to execute in place of this variable identifier[br][br]
+## Entries may be referred to by key in [member default_build_commands] values.
+@export var default_build_variables: Dictionary
+
+## Commands to include in the exported gamepack's [code]default_build_menu.xml[/code].[br][br]
+## Keys, specified as a [String], define the command name as you want it to appear in Radiant.[br][br]
+## Values represent steps taken within the build command.[br][br]They may be either a [String] or an
```

Comment:
Keys are build options, not commands. Values are commands, which are equivalent to steps but listed as commands in NRC, both in the way that the files are formatted and how they show up in NRC. From the Half-Life default_build_menu.xml:
```xml
<build name="CSG, BSP, VIS -fast">
       <command>[csg] "[MapFile]"</command>
       <command>[bsp] "[MapFile]"</command>
       <command>[vis] -fast "[MapFile]"</command>
  </build>
```
So yeah, build options > commands, not commands > steps. I'm sorry if I sound pedantic, but accuracy is important to me for the docs.

Reply:
It's not really exhausting for me; playing "Editor" isn't difficult. It's also interesting to open up NRC's guts a little, a small change of pace. I mostly get worried about offending by "a thousand cuts".